internal/consumer/dlq.go

```diff
@@ -229,7 +236,18 @@ func NewRetryDLQMultiplexer(retryTopic, dlqTopic DLQ, threshold int64) DLQ {
 // If the message RetryCount is greater than or equal to the retryCountThreshold in the multiplexer,
 // the message will be sent to the retry topic.
 // Else, it will be sent to the dlq topic.
-func (d *dlqMultiplexer) Add(m kafka.Message) error {
+func (d *dlqMultiplexer) Add(m kafka.Message, qtypes ...ErrorQType) error {
```

Comment:
Do we accept multiple qtypes? From the implementation it doesn't, but it's not clear from API

Reply:
`RetryQType` and `DLQQtype`.

Reply:
Note that this is not a public interface.